nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -1030,10 +1030,74 @@ public void doNotSupportAssertThatWhenDisabled() {
             "import java.util.Objects;",
             "import javax.annotation.Nullable;",
             "class Test {",
-            "  private void foo(@Nullable Object o) {",
-            "    com.google.common.truth.Truth.assertThat(o).isNotNull();",
+            "  private void foo(@Nullable Object a, @Nullable Object b, @Nullable Object c) {",
+            "    com.google.common.truth.Truth.assertThat(a).isNotNull();",
```

Comment:
Can we separate the Truth test cases from the JUnit/Hamcrest ones? I think also that way we can properly use imports and make them far clearer/more idiomatic to read.

Reply:
Oh. I see we are doing that for the "enabled" case. I still think it's worth to split the "assert handler disabled" tests too, but not super set on it.

Reply:
Separated the negative tests as well.